AUTHORS

```diff
@@ -2683,5 +2683,171 @@ Michael Wei <mwei@cs.ucsd.edu>
+Yann Hamon <yann.hamon@contentful.com>
+Ben Swinburne <ben.swinburne@gmail.com>
+Colin Prince <col@colinprince.com>
+TJKoury <TJKoury@gmail.com>
+dnlup <dwon.dnl@gmail.com>
+Hang Jiang <jianghangscu@gmail.com>
+Vladislav Kaminsky <wlodzislav@outlook.com>
+Daiki Ihara <sasurau4@gmail.com>
+toshi1127 <toshi.matsumoto.2n@stu.hosei.ac.jp>
+nd-02110114 <nd.12021218@gmail.com>
+dkundel <dominik.kundel@gmail.com>
+Evan Plaice <evanplaice@gmail.com>
+simon3000 <simon300000@users.noreply.github.com>
+Marcos Casagrande <marcoscvp90@gmail.com>
+Ruwan Geeganage <rpgeeg@gmail.com>
+Maël Nison <mael@fb.com>
+Gerson Niño <meta.author@gersonnino.me>
+freestraws <freestraws1@gmail.com>
+Daniel Beckert <drbeckert@gmail.com>
+Rivaldo Junior <jrcavalcantejr@gmail.com>
+Rongjian Zhang <pd4d10@gmail.com>
+tonyhty <tonyhty@outlook.com>
+jyjunyz <jyjunyz@163.com>
+tongshouyu <tongshouyu@bytedance.com>
+lixin.atom <lixin.atom@bytedance.com>
+luoyu <luoyu@bytedance.com>
+xinyulee <lixinyu1994123@gmail.com>
+hardfist <1562502418@qq.com>
+shenchen <shenchen@bytedance.com>
+zhoujiamin <zhoujiamin@bytedance.com>
+Chenxi Yuan <yuanchenxi95@gmail.com>
+nilianzhu <nilianzhu@bytedance.com>
+wuchenkai <wuchenkai@bytedance.com>
+xuqinggang <xuqinggang@bytedance.com>
+XGHeaven <xgheaven@gmail.com>
+sinoon <sinoon1218@gmail.com>
+Yaphet Ye <tsyeyuanfeng@126.com>
+OneNail <OneNail@yeah.net>
+陈健 <chenjian.bzh@bytedance.com>
+heben <heben@bytedance.com>
+sujunfei <sujunfei@bytedance.com>
+imhype <543717080@qq.com>
+ptaylor <paul.e.taylor@me.com>
+Boxuan Li <liboxuan@connect.hku.hk>
+Aditya Pratap Singh <adisinghrajput@gmail.com>
+Eugene Ostroukhov <eostroukhov@netflix.com>
+Preveen Padmanabhan <dxb1230@gmail.com>
+Benjamin Ki <me@benjaminki.com>
+Daniel Nalborczyk <dnalborczyk@gmail.com>
+Alba Mendez <me@alba.sh>
+zero1five <zerodengyin@gmail.com>
+Gaelan <gbs@canishe.com>
+Jacob <jacobq@gmail.com>
+himself65 <himself65@outlook.com>
+Dan Beglin <dbeglinuk@gmail.com>
+Anish Asrani <anishasrani@gmail.com>
+teams2ua <teams2ua@gmail.com>
+oksana <ok.semonenko@gmail.com>
+Grigorii K. Shartsev <me@shgk.me>
+Kopachyov Vitaliy <kopachyov.vitaliy@yandex.ru>
+MurkyMeow <dinosowermurky@gmail.com>
+Evgenii Shchepotev <evgenii.schepotiev@gmail.com>
+martyns0n <zogacc@gmail.com>
+Levin Eugene <lzhnek@gmail.com>
+Alexander Avakov <yaavakov@gmail.com>
+Grigory Gorshkov <petralmazov100@gmail.com>
+Keroosha <mr.dead.toast@gmail.com>
+Tariq Ramlall <srcmake@gmail.com>
+Alex Pry <opterione@gmail.com>
+Yuriy Vasiyarov <yvasiyarov@ozon.travel>
+Mikhail Kuklin <mihan007@ya.ru>
+went.out <went.out@gmail.com>
+Kyle Zhang <icese7en@gmail.com>
+Alex Temny <dashkamimicry100@gmail.com>
+Alex Aubuchon <alex@aub.dev>
+Samuel Attard <samuel.r.attard@gmail.com>
+rexagod <rexagod@gmail.com>
+Antonio Kukas <tonykukas@gmail.com>
+murgatroid99 <mlumish@google.com>
+Saagar Jha <saagar@saagarjha.com>
+vmarchaud <contact@vmarchaud.fr>
+Milad Farazmand <miladfar@ca.ibm.com>
+mutao <mutao-hf@loongson.cn>
+= <=>
```

Comment:
@ssample812 If you’d like to be listed in the authors list with some kind of name other than this, please let us know.

Reply:
Samantha Sample <ssample812@gmail.com>

Thank you!

Reply:
Oh, didn’t see that you had another entry here already. Done!